packages/client/components/ArchiveTaskRoot.tsx

```diff
@@ -17,24 +17,24 @@ const query = graphql`
 `
 
 export interface ArchiveTaskRootProps {
-  teamId?: string
+  teamIds?: string[] | null
+  userIds?: string[] | null
   team?: any
-  showHeader?: boolean
+  returnToTeamId?: string
```

Comment:
`teamId` and `showHeader` are mutually inclusive, so I combined them into 1 variable to clean this up a bit